src/windows/stumpless.def

```diff
@@ -187,3 +187,7 @@ EXPORTS
   stumplog_str                                  @172
   stumplog_trace_str                            @173
   stumpless_get_facility_string                 @174
+
+; added in v2.1.0
```

Comment:
you can remove the blank space and comment - this entire block of functions is new to v2.1.0